src/commands/veloce/debug/push.ts

```diff
@@ -50,11 +51,63 @@ export default class Org extends DebugSfdxCommand {
 
     const memberMap = splitMembers(members);
 
+    await this.sendModel(debugSession, rootPath, memberMap['model']);
     await this.sendDrools(debugSession, rootPath, memberMap['drl']);
     // Return an object to be displayed with --json
     return {};
   }
 
+  private findAllModels(sourcepath: string): Set<string> {
+    const result = new Set<string>();
+    const filenames = readdirSync(`${sourcepath}/model`);
+    filenames.forEach((file) => {
+      result.add(file);
+    });
+    return result;
+  }
+
+  private async sendModel(debugSession: DebugSessionInfo, rootPath: string, member: Member) {
+    if (!member) {
+      return;
+    }
+    const params = {
+      veloceNamespace: '',
+      instanceUrl: `${debugSession.instanceUrl}`,
+      organizationId: `${debugSession.orgId}`,
+      oAuthHeaderValue: 'Dummy',
+    };
+    const authorization = Buffer.from(JSON.stringify(params)).toString('base64');
+    const headers = {
+      'dev-token': debugSession.token,
+      Authorization: authorization,
+      'Content-Type': 'application/json',
+    };
+    const backendUrl: string | undefined = debugSession.backendUrl;
+    const models = this.findAllModels(rootPath);
+    for (const name of models) {
+      if (member.all || member.names.includes(name)) {
+        // load PML
+        const pml = readFileSync(`${rootPath}/model/${name}/${name}.pml`, 'utf8').toString();
+        try {
+          await axios.post(
+            `${backendUrl}/services/dev-override/model/${name}/pml`,
+            { content: pml },
+            {
+              headers,
+            },
+          );
+        } catch (error) {
+          if (error instanceof AxiosError) {
+            this.ux.log(`Failed to deploy ${name}: ${error.response?.data} code ${error.response?.status}`);
+          } else {
+            this.ux.log(`Failed to deploy ${name}: ${JSON.stringify(error)}`);
+          }
+        }
+        this.ux.log('PML Successfully Loaded!');
+      }
+    }
+  }
```

Comment:
This could be run in parallel. Please leave a TODO comment for improvement.

Reply:
added todo, however not sure thsi will be slow even without parallelizing, since it is simply in-memory sprling controller